src/utils.js

```diff
@@ -31,7 +32,9 @@ export const defaultOptions = {
 export function clear (forceClear = false, options) {
   resetCache()
   if (forceClear || options.clearConsoleOnUpdate) {
-    console.clear()
     resetLastNotification()
+    if (options.allowConsoleClears) {
+      console.clear()
+    }
```

Comment:
I wasn't sure exactly what `resetLastNotification` does. Not sure if it makes more sense for it to be inside the `if (options.allowConsoleClears)` block or not?